test/internal/helpers/Player.java

```diff
@@ -44,6 +44,20 @@
 import org.mockito.Mockito;
 
 public class Player {
+
+  private static void addItemRequirement(Cleanups cleanups, AdventureResult item) {
+    String requirement = EquipmentDatabase.getEquipRequirement(item.getItemId());
```

Comment:
I would probably rename this to `withStatsRequiredForEquipment`, remove the import of `cleanups` and `add` it to the cleanups in `withEquipped`

Reply:
OK. Done.